libs/template/spec/render-strategies/render-strategies/native.strategy.spec.ts

```diff
@@ -0,0 +1,82 @@
+// tslint:disable-next-line:nx-enforce-module-boundaries
+import { mockConsole } from '@test-helpers';
+import { getStrategies } from '@rx-angular/template';
+import { getMockNativeStrategyConfig } from '../../fixtures';
+import { from, of } from 'rxjs';
+import createSpy = jasmine.createSpy;
+
```

Comment:
```suggestion

/**
 * NATIVE STRATEGY
 * Doing the same thing as Angular natively. Based on markForCheck.
 */
 
```

Reply:
Comments will just help to understand what change detection mechanism strategy is based on.